src/backend/utils/misc/guc_gp.c

```diff
@@ -4932,7 +4935,7 @@ struct config_enum ConfigureNamesEnum_gp[] =
 		},
 		&Gp_interconnect_type,
 		INTERCONNECT_TYPE_UDPIFC, gp_interconnect_types,
```

Comment:
Is gp_interconnect_types a fixed array? It should be consistent with the registered ic.

Reply:
fix it with [commit](https://github.com/cloudberrydb/cloudberrydb/pull/646/commits/cc3a12c740074a515a04896ea27a63cdf46ace57)